src/lib/app.js

```diff
@@ -116,6 +116,8 @@ class App {
 
     router.get('/transfers/:id', transfers.getResource)
     router.get('/transfers/:id/state', transfers.getStateResource)
+    router.get('/transfers/:id/hash_preimage', transfers.getPreimage)
```

Comment:
No one besides the ledger is supposed to know the preimage until the condition is met, at which point the ledger attaches it as the fulfillment.